centreon/www/class/centreonLDAP.class.php

```diff
@@ -371,7 +371,7 @@ public function findUserDn($username): ?string
      * @param string $group The group
      * @return string|bool The dn string or false if not found
      */
-    public function findGroupDn($group): ?string
+    public function findGroupDn($group): string|bool
```

Comment:
In PHP 8.2 you can type with true or false

```suggestion
    * @return string|false The dn string or false if not found
    */
    public function findGroupDn($group): string|false
```